packages/walletconnect-connector/src/WalletConnectConnector.ts

```diff
@@ -75,7 +75,7 @@ export class WalletConnectConnector extends FuelConnector {
 
   async configProviders(config: WalletConnectConfig = {}) {
     this.config = Object.assign(config, {
-      fuelProvider: config.fuelProvider || await FuelProvider.create(TESTNET_URL),
+      fuelProvider: config.fuelProvider || FuelProvider.create(TESTNET_URL),
```

Comment:
This `create` method, returns a promise, but it isn't awaited, somehow, this way, the test pass

Any idea why?

Reply:
this await is needed to have a fuelProvider returning.. otherwise `fuelProvider` will be a promise on that case